dotCMS/src/integration-test/java/com/dotmarketing/portlets/workflows/business/WorkflowAPITest.java

```diff
@@ -1914,6 +1923,119 @@ public void validatingNoObligatorieContentTypeWorkflow()
         }
     }
 
+    /**
+     * Validate that when a Content type is modified associating and/or removing workflows schemes
+     * then the existing workflow tasks associated to the scheme keep the current status and the
+     * removed ones are set on null
+     *
+     * @throws DotDataException
+     */
+    @Test
+    public void saveScheme_keepExistingContentWorkflowTaskStatus_IfWorkflowSchemeRemainsAssociated()
```

Comment:
This method contains too many lines, which will make it difficult to maintain. Can this be refactored? eg. : divide responsibility using private methods or in case several things are being tested in the same method, split them in other tests